servicemesher/client/pom.xml

```diff
@@ -10,11 +10,12 @@
 	<version>1.00</version>
 
 	<properties>
-		<java.version>1.8</java.version>
+		<java.version>11</java.version>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<version.light-4j>1.6.2</version.light-4j>
-		<version.light-rest-4j>1.6.2</version.light-rest-4j>
-		<version.openapi-parser>1.6.2</version.openapi-parser>
+		<project.main.class>com.networknt.server.Server</project.main.class>
+		<version.light-4j>2.0.2-SNAPSHOT</version.light-4j>
+		<version.light-rest-4j>2.0.2-SNAPSHOT</version.light-rest-4j>
```

Comment:
The latest light-bot configuration only recognizes version.light-4j. As light-rest-4j and openapi-parser have the same version, let's just use the version.light-4j so that light-bot can upgrade it automatically for each release.

Reply:
change the POM to use light-4j version only